build.gradle

```diff
@@ -76,8 +76,8 @@ allprojects {
   }
 
   bintray {
-    user = project.hasProperty('bintrayUser') ? project.getProperty('bintrayUser') : null
-    key  = project.hasProperty('bintrayKey')  ? project.getProperty('bintrayKey')  : null
+    user = System.env.BINTRAY_USER ?: null
```

Comment:
Being able to set it with a property is useful. Also I wonder how many know about the Elvis operator. Readability trumps conciseness imho.
I'd say:
`user = System.env.BINTRAY_USER ? System.env.BINTRAY_USER : project.getProperty('bintrayUser')`